app/views/builds/show.html.haml

```diff
@@ -20,9 +20,23 @@
     %dd
       = build_status(@build.overall_status)
 
+    -if @build.started_at
+      %dt Overall Duration
+      %dd= build_duration(@build.started_at, @build.overall_completed_at)
+
     %dt Build Status
     %dd= build_status(@build.status)
 
+    %dt Started
+    %dd= @build.started_at || "N/A"
+
+    %dt Completed
+    %dd= @build.completed_at || "N/A"
+
+    -if @build.started_at
```

Comment:
Is there an advantage here to the if conditional for both overall duration and duration? I think having a consistent table might outweigh the ugliness of seeing  'N/A' in the output.